torch/cuda/__init__.py

```diff
@@ -260,7 +260,7 @@ def _check_capability():
     CUDA_ARCHES_SUPPORTED = {
         "12.6": {"min": 50, "max": 90},
         "12.8": {"min": 70, "max": 120},
-        "13.0": {"min": 75, "max": 120},
+        "13.0": {"min": 75, "max": 121},
```

Comment:
These warning are also incorrect if a +PTX target is specified, right: since those are forward compatible? Something to think about for another PR I guess.